frontend/src/App.vue

```diff
@@ -6,17 +6,18 @@
       :height="64"
       :width="64"
       :color="isLoadingColor"
-      background-color="transparent" 
+      background-color="transparent"
       loader="spinner"
     ></loading>
     <div>
-      <v-card-title>{{messages.title}}</v-card-title>
-      <v-card-subtitle>{{messages.description}}</v-card-subtitle>
+      <v-card-title>{{ messages.title }}</v-card-title>
```

Comment:
try to avoid making formatting changes in unmodified code.
It makes reviewing harder...